assets/js/state/sagas/notifications.jsx

```diff
@@ -2,16 +2,40 @@
 import React from 'react';
 import { takeEvery } from 'redux-saga/effects';
 import { toast } from 'react-hot-toast';
-import { NOTIFICATION } from '@state/notifications';
+import {
+  NOTIFICATION,
+  DISMISSABLE_NOTIFICATION,
+  DISMISS_NOTIFICATION,
+} from '@state/notifications';
+import DismissableToast from '@common/DismissableToast';
 
 export function* notification({ payload }) {
-  const { text, icon } = payload;
+  const { text, icon, id, duration } = payload;
   toast(<p className="text-sm font-medium text-gray-900">{text}</p>, {
     position: 'top-right',
     icon,
+    id,
+    duration: duration || 4000,
   });
 }
 
+export function* dismissableNotification({ payload }) {
+  const { text, icon, id, duration } = payload;
+  toast((t) => <DismissableToast text={text} toastInstance={t} />, {
+    position: 'top-right',
+    icon,
+    id,
+    duration: duration || 4000,
+  });
+}
+
+export function* dismissNotification({ payload }) {
+  const { notificationID } = payload;
```

Comment:
ocd: every other notification we use `id` and `notificationID` here hehe

Reply:
done